.gitignore

```diff
@@ -7,3 +7,4 @@ doc/html/*
 doc/polymorphic_value_reference.xml
 makePDF.sh
 talks/.ipynb_checkpoints
+_deps
```

Comment:
The "_deps" folder should be created in the build directory, not at the root.  Should this not be?
```
**/_deps
```

Reply:
Removed.

This should be in another PR.